uiServices/services_test.go

```diff
@@ -71,7 +109,7 @@ func TestValidJsonPost(t *testing.T) {
 		t.Error(err)
 	}
 
-	if http.StatusCreated != 201 {
+	if w.Code == http.StatusCreated  {
```

Comment:
Check for != http.statusOK. You want to make sure any status code except your valid one fails the test.

Also decide whether you're sending down statusOK or statusCreated when the request is successful.